backend/common/utils/cxg_generation_utils.py

```diff
@@ -31,6 +34,31 @@ def convert_dictionary_to_cxg_group(cxg_container, metadata_dict, group_metadata
             metadata_array.meta[key] = value
 
 
+def convert_uns_to_cxg_group(cxg_container, metadata_dict, group_metadata_name="uns", ctx=None):
+    """
+    Convert uns (unstructured) metadata to CXG output directory specified
+    Generate deep zoom assets for spatial data
+    """
+
+    spatial_processor = SpatialDataProcessor()
+
+    array_name = f"{cxg_container}/{group_metadata_name}"
+    object_filtered = {}
+
+    tiledb.from_numpy(array_name, np.zeros((1,)))
+
+    with tiledb.open(array_name, mode="w", ctx=ctx) as metadata_array:
+        for key, value in metadata_dict.items():
+            if key == UNS_SPATIAL_KEY:
+                for object_id, content in value.items():
+                    object_filtered = spatial_processor.filter_spatial_data(content, object_id)
+                    spatial_processor.create_deep_zoom_assets(cxg_container, content)
+            else:
+                continue
+
+            metadata_array.meta[key] = pickle.dumps(object_filtered)
```

Comment:
nit: drop the `else: continue` and move `metadata_array.meta[key] = pickle.dumps(object_filtered)` in side `if key == UNS_SPATIAL_KEY:`